frontend/src/metabase-types/api/search.ts

```diff
@@ -49,7 +49,7 @@ export interface SearchResults {
 }
 
 export interface SearchResult {
-  id: number | undefined;
+  id: number;
```

Comment:
@iethree This line was introduced in https://github.com/metabase/metabase/pull/31783/files#diff-2a8c70a6c0fc73f1996c801f0bf92ed98a2e09ca43f102d93f6cb0c41fb66e81R50. And could be `undefined`. Do you know if it's valid, or should every search result have an ID? I feel like every search result should have an ID because we're picking an entity out of a database.